### ISSUE TITLE
potential controlled terms additions

### ISSUE THREAD

**Issue by lzehl** [HumanBrainProject/openMINDS_controlledTerms#101]
How about these new TissueSampleAttributes (?):
+ sliced
+ untreated
+ mounted
+ fixated

And this new TissueSampleType (?):
+ tissue slice series

And this new SubjectAttribute (?):
+ comatose (OR unconscious)

**Comment by UlrikeS91** [comment]
> How about these new TissueSampleAttributes (?):
> 
> * sliced
> * untreated
> * mounted
> * fixated

- sliced: I would say no. We have TissueSampleType "tissue slice" and I cannot come up with a use case were any other "TissueSampleType" would make sense in combination with sliced as "TissueSampleAttribute", maybe "nerve" + "sliced". But then I would rather introduce a new TSType...
- untreated: ok
- mounted: ok, maybe add "free floating" or similar in parallel?
- fixated: ok

On a general note: What is our perspective towards these atributes? As many details as possible or only crucial information? I'm asking because of terms like "mounted". It absolutely is a TSAttribute, but when would this information ever be crucial? It is procedural information which could also be captured in the Protocol/ProtocolExecutions. But then again all these attributes probably fall under this category, right?   
 
> And this new TissueSampleType (?):
> 
> * tissue slice series

I'm leaning towards no. TissueSampleCollections give part of this information already - not all of it, though. Could you describe what use case you had in mind? I find "series" is a bit vague, or was that on purpose? In the broadest sense "series" just means that they have something in common. With my background, I would assume tissue slices that were collected sequentially with equal spacing between the collected slices. If this is was you were aiming for, we may want to change the name slightly and it will need a good definition. 
Question is still if we want to go into this level of detail? So far, we kept TSTypes it quite high-level. I'm not against more detail, just putting the question out there ;)   


> And this new SubjectAttribute (?):
> 
> * comatose (OR unconscious)

I think "unconscious" might be misused, and there are still huge debates about what can be considered "(un)conscious". If "comatose" is related to a specific use case, where a patient actually is in a coma, we would colloquially use "unconscious" because we can't say whether or not that person is still self-aware, but maybe they are just "unresponsive" and unable to show that they are self-aware (use-case: locked-in syndrome). 
We also have "anaesthetized" which is more descriptive of the state the subject is in. We would assume that they would be unconscious then, but we don't know for certain. 
Long story short: I would add "comatose". It's more descriptive to the state the subject is in and less loaded with assumptions - similarly to "anaesthetized".

**Comment by lzehl** [comment]
For the attributes question: I think what is crucial depends on the use case. You want to use attributes primarily to distinguish states from each other that do not differ in their main properties.

For the TissueSampleType suggestion: I think type "tissue slice" for a collection sounds/is wrong. A respective display would need to be always manipulated.

I agree with the rest. Thanks for the feedback :) are you taking over ?

**Comment by UlrikeS91** [comment]
> For the attributes question: I think what is crucial depends on the use case. You want to use attributes primarily to distinguish states from each other that do not differ in their main properties.

Yeah, true. Good point. 

> For the TissueSampleType suggestion: I think type "tissue slice" for a collection sounds/is wrong. A respective display would need to be always manipulated.

Yes, I agree. It will look wrong but the type is still correct - it's just the quantity that is not. But this will be a problem for all the TS vs. TSCollections, right? So, manipulating the respective display would be the best solution anyway :) 

> I agree with the rest. Thanks for the feedback :) are you taking over ?

Great :) Yes, I can add them. 

Summary (mostly for myself):
NOT adding:
- sliced
- tissue slice series

Adding as requested:
- untreated
- mounted
- fixated
- comatose

Adding additionally:
- free floating (as counterpart to mounted)

**Comment by lzehl** [comment]
fine for leaving the TSType for now. @olinux be aware that this will require display manipulations
